src/components/Episodes/Episodes.jsx

```diff
@@ -8,23 +9,20 @@ import {
 
 export default function Episodes(){
 
-	const pagination = data.map(RENDER.pagination);
 	const episodes = data.map(RENDER.episode);
 
 	return (
-		<article>
-			<EpisodesProvider>
-				<ol>
-					{ episodes }
-				</ol>
-				<nav>
-					<button aria-label="Previous episode." />
-					<button aria-label="Next episode." />
+		<>
+			<article>
+				<EpisodesProvider>
```

Comment:
SO, we need `<EpisodesProvider>` to wrap _both_ the `episodes` _and_ the `<EpisodesControls>`, if we don't then we're creating two instances of the component, and thus there'll be two separate states with `activeIndex` being provided.

Reply:
Cool, had a feeling that would come back to me!
Wasn't sure about the nav being in the article, but it definitely seemed weird as sibling when it meant adding provider again.